istio.deps

```diff
@@ -11,6 +11,6 @@
     "name": "CNI_REPO_SHA",
     "repoName": "cni",
     "file": "",
-    "lastStableSHA": "5f438d5ab98dd1f97ff7c0f5e437e044c314a9c3"
+    "lastStableSHA": "787a566773980c28bb3cffd54af3fb6256031c5a"
```

Comment:
/wait

Reply:
@mandarjog are you going to fix this and update proxy repo instead? Also, could you pick up the most recent commit?